src/textdescriptives/sklearn_featurizer.py

```diff
@@ -0,0 +1,107 @@
+from typing import Iterable, List, Optional
+
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
```

Comment:
this will require us to have this as a dependency - maybe we should create an error here.
```
try:
   import sklearn ...
except importerror as e:
   raise ValueError("Tried to import sklearn, if you want to use it with textdescriptives install it using pip install 'textdescriptives[sklearn]'")

Reply:
potentially move this script into a folder called integrations?